src/commcare_cloud/ansible/roles/pg_repack/tasks/main.yml

```diff
@@ -62,23 +80,24 @@
     group: "postgres"
     mode: 0755
 
+- name: Copy pg_repack execution script
+  become: yes
+  template:
+    src: "pg_repack_cron.sh.j2"
+    dest: "/usr/local/sbin/pg_repack_cron.sh"
```

Comment:
you can't use the same filename for each job otherwise the file will get overwritten each time (this is executed in a loop).

Reply:
Noted Simon. thank you.